src/main/java/frc/robot/commands/StartShooterCommandGroup.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandGroupBase;
+import frc.robot.subsystems.BallShooterSubsystem;
+import frc.robot.subsystems.BallFeederSubsystem;
+
+public class StartShooterCommandGroup extends CommandGroupBase {
+
+    public StartShooterCommandGroup(BallShooterSubsystem subsystem, BallFeederSubsystem subsystem)
+    {
+        addSequential(new RunBallFeeder());
+        addSequential(new RunBallShooter());
+
+    }
+
+   
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Ok, Patrick. This is enough space for drunk Ryley AND the entire plane. (You can just delete all the empty space there, before it turns into a black hole and swallows up our entire existence.)